test/test_connection.py

```diff
@@ -153,6 +150,14 @@ def test_ping_pong(self, as_client):
             data = wire_data[2:]
         assert data == payload
 
+        # Verify that connection vends the PongReceive event with the correct payload
```

Comment:
Is "vend" not the correct verb to use here? If I used "sends" who would it be sending it to?